app/presenters/account_show_presenter.rb

```diff
@@ -30,14 +41,49 @@ def show_service_provider_continue_partial?
     sp_name.present? && sp_session_request_url.present?
   end
 
-  def show_gpo_partial?
-    user.gpo_verification_pending_profile?
+  def showing_alerts?
+    show_service_provider_continue_partial? ||
+      show_password_reset_partial?
   end
 
-  def showing_any_partials?
-    show_service_provider_continue_partial? ||
-      show_password_reset_partial? ||
-      show_gpo_partial?
+  def active_profile?
+    return @active_profile if defined?(@active_profile)
+    @active_profile ||= user.active_profile.present?
```

Comment:
the early return means the `||=` is redundant, we can do a direct assign
```suggestion
    return @active_profile if defined?(@active_profile)
    @active_profile = user.active_profile.present?
```


but also, the `user.active_profile` is already memoized, so if we're worried about saving a trip to the DB, this is not necessary

```suggestion
    user.active_profile.present?
```